### ISSUE TITLE
Aetherworks seems to conflict with Buildcrafts oil textures

### ISSUE THREAD

**Issue by catwet12** [V0idWa1k3r/Aetherworks#3]
It would seem that your mod is conflicting with buildcrafts oil textures and this problem has occured for others as this problem is logged on buildcrafts issues but not yours due to users missatributing the offending mod. I have found that only when your mod (AetherWorks-1.12.2-1.0.0.1-rc2) is installed into this modlist [modlist.txt](https://github.com/V0idWa1k3r/Aetherworks/files/2017804/modlist.txt) only then do the buildcraft oil textures show as missing.

**Comment by vrork** [comment]
https://github.com/BuildCraft/BuildCraft/issues/4165#issuecomment-390266135

Linking the relevant issue

**Comment by V0idWa1k3r** [comment]
I somehow managed to replicate the issue WITHOUT AetherWorks present in my workspace so I am willing to believe this is an issue of buildcraft and not aetherworks.
Proof:
![2018-05-18_21 34 37](https://user-images.githubusercontent.com/23249796/40251819-13907708-5ae3-11e8-9d79-2465c9cdec87.png)
![2018-05-18_21 34 48](https://user-images.githubusercontent.com/23249796/40251820-13ae438c-5ae3-11e8-8ff9-919174e656ae.png)
I am leaving this issue open as it seems that my mod is indeed somehow 'enabling' it to happen. I am waiting for futher action from either buildcraft or my testing. For now I am unable to do much as even disabling the entire mod up to the point of deleting it entirely still leaves the issue present.
I'll still be looking into this further.

**Comment by vrork** [comment]
Alright. Weird indeed.

**Comment by vrork** [comment]
Last update, still breaking the textures.

**Comment by V0idWa1k3r** [comment]
Yeah, I figured that if deleting the entire mod doesn't help what I did also won't help. I am not sure what the cause of issue is and I believe that it's BC that should trace it. AW fluids don't conflict with BC fluids in any way and the only texture stiching that AW does is adding runes to the map(so it's not fluid related in any way either).

**Comment by vrork** [comment]
The weird thing is that if i add your and BC, the textures break. If i remove yours, they are fine.., it makes no sense...
